SitumWayfinding/Classes/Helpers/UIUtils.swift

```diff
@@ -46,6 +46,43 @@ extension UIView {
         self.bottomAnchor.constraint(equalTo: superview.bottomAnchor, constant: 0).isActive = true
         self.leadingAnchor.constraint(equalTo: superview.leadingAnchor, constant: 0).isActive = true
         self.trailingAnchor.constraint(equalTo: superview.trailingAnchor, constant: 0).isActive = true
-        
+    }
+    
+    func roundCorners(corners: UIRectCorner) {
+        let radius: CGFloat = 5
+        if #available(iOS 11.0, *) {
+            self.layer.maskedCorners = rectCornerToMaskedCorners(corners: corners)
+            self.layer.cornerRadius = radius
+        } else {
+            let path = UIBezierPath(roundedRect: self.bounds,
+                byRoundingCorners: corners,
+                cornerRadii: CGSize(width: radius, height: radius))
+            let mask = CAShapeLayer()
+            mask.path = path.cgPath
+            self.layer.mask = mask
+        }
+    }
+    
```

Comment:
Podes deixar un comentario aquí do tipo para redondear os bordes é máis doado traballlar cos CACornerMAsk e por eso se fai esa traducción. Se non de primeiras non é moi doado de entender